src/content/docs/logs/forward-logs/heroku-log-forwarding.mdx

```diff
@@ -13,18 +18,57 @@ import addData from 'images/add-data.png'
 
 import herokuTile from 'images/heroku-tile.png'
```

Comment:
I'll prefer to get rid of them. I think they don't look well on the docs page, and also it's confusing because Night/Day mode and the colors of the images. Thanks!